core/remoteviewserver.cpp

```diff
@@ -91,7 +93,6 @@ bool RemoteViewServer::isActive() const
 
 void RemoteViewServer::sendFrame(const RemoteViewFrame &frame)
 {
-    m_clientReady = false;
```

Comment:
But this was correct, no? If the source changed after we grabbed the frame we eventually need to get a second frame. After all we want to show the correct state in the client.